docs/releases/3.2.txt

```diff
@@ -53,6 +53,16 @@ needed. As a consequence, it's deprecated.
 
 See :ref:`configuring-applications-ref` for full details.
 
+Expressions alias
+-----------------
+
+Expressions that don't need to be selected can now be saved as an alias for
+later reuse with other ``QuerySet`` methods via a new
+:meth:`~django.db.models.query.QuerySet.alias` method. This is useful when
+the result of the expression itself is not needed but is used for filtering,
+ordering, or as a part of a complex expression. Not selecting the unused value
+removes redundant work from the database which should result in better performance.
+
```

Comment:
It's not a major feature, IMO, I would move it to the `Minor features -> Models`, maybe:
```
* The new :meth:`.QuerySet.alias` method allows creating reusable aliases for
  expressions that don't need to be selected but are useful for filtering,
  ordering, or as a part of a complex expression.
```

Reply:
I would argue that adding a new public `QuerySet` method can be considered a major feature. As far as I can understand, minor features are generally amendments to existing functionality like behavior change or introducing new parameters. But obviously I don't have enough experience to make a decision. What would you say?

Reply:
> minor features are generally amendments to existing functionality like behavior change or introducing new parameters

Not really, e.g. [`QuerySet.bulk_update()`](https://docs.djangoproject.com/en/3.0/releases/2.2/#models) or almost all new database functions are in minor features. We can decide to promote it before the 3.2 release but for now I would add it to the minor features.